opensanctions/crawlers/md_interdictie.py

```diff
@@ -37,6 +46,24 @@
 
 COUNTRY = "md"
 
+# Fondator/Administrator – 
+# or
+# Fondator - TOLMAŢCHI VALERI Administrator - TOLMAŢCHI VALERI
+# or
+# Lista conducătorilor - Galin Anatolie, Lista fondatorilor - Galin Anatolie
+# (List of leaders)
+# or
+# Conducători -
+# (Leaders)
+ROLES = { 
```

Comment:
FWIW, we're also getting the BO info from `md_companies`, so I'm not 100% sure we should go through the effort of parsing the ownership info here.

Reply:
we said we'd keep the ownerships since they're done.

I added memberships for the rest but not adding their role now, since the way I'm scraping them doesn't leave me with the romanian term - they all become UNKNOWN or ADMINISTRATOR when they might be leader, etc.

Is that cool?